### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by Martlet-Tech** [romainreignier/minimal_raspberrypi_buildroot#1]
ln -sf rootfs.ext2 /home/zhaitao/build/images/rootfs.ext4
>>>   Executing post-image script board/raspberrypi3/post-image.sh
INFO: cmd: "mkdir -p "/home/zhaitao/build/build/genimage.tmp"" (stderr):
INFO: cmd: "rm -rf "/home/zhaitao/build/build/genimage.tmp"/*" (stderr):
INFO: cmd: "mkdir -p "/home/zhaitao/build/build/genimage.tmp"" (stderr):
INFO: cmd: "cp -a "/home/zhaitao/build/target" "/home/zhaitao/build/build/genimage.tmp/root"" (stderr):
INFO: cmd: "find '/home/zhaitao/build/build/genimage.tmp/root' -depth -type d -printf '%P\0' | xargs -0 -I {} touch -r '/home/zhaitao/build/target/{}' '/home/zhaitao/build/build/genimage.tmp/root/{}'" (stderr):
ERROR: file(bcm2710-rpi-3-b-plus.dtb): stat(/home/zhaitao/build/images/bcm2710-rpi-3-b-plus.dtb) failed: No such file or directory
ERROR: vfat(boot.vfat): could not setup bcm2710-rpi-3-b-plus.dtb
Makefile:814: recipe for target 'target-post-image' failed
make[1]: *** [target-post-image] Error 1
Makefile:23: recipe for target '_all' failed
make: *** [_all] Error 2

**Comment by romainreignier** [comment]
Indeed, the RPi3B+ is not supported yet.
All you have to do is to add the device tree file here: https://github.com/romainreignier/minimal_raspberrypi_buildroot/blob/95210df25118ce561b3005610a5472ac44214430/configs/raspberrypi3_minimal_defconfig#L33

**Comment by Martlet-Tech** [comment]
![image](https://user-images.githubusercontent.com/4844538/60560158-5beb4c80-9d81-11e9-9967-24c9d10092a9.png)
sorry, I don't understand what you mean, this line of code is already here.

**Comment by romainreignier** [comment]
Ah, sorry if you misunderstood.

Your error is:

> ERROR: file(bcm2710-rpi-3-b-plus.dtb): stat(/home/zhaitao/build/images/bcm2710-rpi-3-b-plus.dtb) failed: No such file or directory

So the file `bcm2710-rpi-3-b-plus.dtb` is missing. Which is normal because it is not included in the config as the others.

So you should add it to the config file as it was done on upstream buildroot, see here:
https://github.com/buildroot/buildroot/blob/ad125248030dcb7b3a8e13c96b727262799c48b9/configs/raspberrypi3_defconfig#L19

But note that you should update the kernel too because my config uses an old 4.9 kernel while the 3B+ has been added in 4.14.

So basically, on that line:
https://github.com/romainreignier/minimal_raspberrypi_buildroot/blob/95210df25118ce561b3005610a5472ac44214430/configs/raspberrypi3_minimal_defconfig#L28

Replace `240d0482ad9341f7d98b06705d0a55e298034121` by `5eeff139ea9135db6e01a58ef613338f1d0899d8`.

And maybe the toolchain would need to be updated.

I will try to update the config but I do not own any 3B+ to test it.

**Comment by romainreignier** [comment]
Please, try the `update_3b+` branch:
https://github.com/romainreignier/minimal_raspberrypi_buildroot/tree/update_3b+

**Comment by Martlet-Tech** [comment]
worked for my rpi3, 
thank you so much for your help!
and another problem, what is the user name / passwd to login?
![image](https://user-images.githubusercontent.com/4844538/60641367-13559100-9e5e-11e9-9641-35941f0bbc6f.png)

**Comment by romainreignier** [comment]
Ok, great, I will merge this branch to master.

The login is `root` and here is no password (Buildroot defaults).
